src/utils/compositionUtils.js

```diff
@@ -186,7 +193,7 @@ function buildGriddleReducerObject(reducerObjects) {
   if (reducerObjects.length > 0) {
     // remove the hooks and extend the object
     for(const key in reducerObjects) {
-      const reducer = reducerObjects[key];
+      const reducer = pickBy(reducerObjects[key], (value, key) => isString(key));
```

Comment:
reducer is gotten by direct index still, problem is when default reducers are imported in  [src/core](https://github.com/GriddleGriddle/Griddle/blob/master/src/core/index.js#L2) reducer object has extra Symbol property (which I think it wasn't there before) and this extra Symbol is causing issues in `src/utils/compositionUtils.js` since reducer object keys are expected to be string.

Reply:
Ah sorry, I see now you are stripping out the nonString items.  So I think this is good,
yet I wonder where the non-string items are coming from.